scripts/set_kokkos_env.sh

```diff
@@ -0,0 +1,3 @@
+export MPICH_CXX=/home/uy7/software/Trilinos/packages/kokkos/config/nvcc_wrapper
```

Comment:
Would be better if you add `nvcc_wrapper` in your `PATH`, and then simply do
```
export MPICH_CXX=nvcc_wrapper
```
This way this could env could be used by somebody else.

Also add
```
export OMPI_MPICXX=nvcc_wrapper
```